pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -196,11 +196,11 @@ func (m *manager) GetMemoryNUMANodes(pod *v1.Pod, container *v1.Container) sets.
 	}
 
 	if numaNodes.Len() == 0 {
-		klog.V(5).Infof("No allocation is available for (Pod: %s, Container: %s)", pod.Name, container.Name)
+		klog.V(5).InfoS("No allocation is available", "podName", pod.Name, "container", container.Name)
```

Comment:
We have the full Pod spec available, let's use the KObj

```suggestion
		klog.V(5).InfoS("No allocation is available", "pod", klog.KObj(pod), "container", container.Name)
```